com.obsproject.Studio.Plugin.Kinect.yaml

```diff
@@ -0,0 +1,58 @@
+id: com.obsproject.Studio.Plugin.Kinect
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/Kinect
+  prepend-pkg-config-path: /app/plugins/Kinect/lib/pkgconfig # Needed for libfreenect2
+cleanup:
+  - /lib/Kinect/*.a
+modules:
+  - shared-modules/libusb/libusb.json
+
+  - name: libfreenect
+    buildsystem: cmake-ninja
+    cleanup:
+      - /lib/pkgconfig
+      - /lib/fakenect
+      - /include
+      - /bin
+    sources:
+      - type: git
+        url: https://github.com/SirLynix/libfreenect.git
+        commit: 5e2cd8e18877fbe7ef4c863161e90873c06a9180
+
+  - name: libfreenect2
+    buildsystem: cmake-ninja
+    cleanup:
+      - /lib/pkgconfig
+      - /lib/cmake
+      - /include
+    sources:
+      - type: archive
+        url: https://github.com/OpenKinect/libfreenect2/archive/refs/tags/v0.2.1.tar.gz
+        sha256: c09e52c97b0e90335f4762ed5363293ad0fc1ea0064e578b879e7d15cc3559df
```

Comment:
Have you considered integrating with [flatpak-external-data-checker](https://github.com/flathub/flatpak-external-data-checker)? It would automatically observe and apply dependency updates to this manifest. I think you can use the [git checker](https://github.com/flathub/flatpak-external-data-checker#git-checker) here.

Up to you, f-e-d-c is more of a convenience than a requirement.

Reply:
I couldn't get it to work with obs-kinect archives from github. So for now, it'll be used with the other two modules, libfreenect and libfreenect2, till I figure out how to use the json checker.